### ISSUE TITLE
Datepicker: Month in german

### ISSUE THREAD

**Issue by michaelbenker** [gcanti/tcomb-form-native#340]
### Version

Tell us which versions you are using:

- tcomb-form-native v0.6.7
- react-native v0.41.2

Hello,

how can I get the Month from the Datepicker in german?
Thanx

**Comment by javiercr** [comment]
The datepicker that comes with the bootstrap templates ([`datepicker.android.js`](https://github.com/gcanti/tcomb-form-native/blob/master/lib/templates/bootstrap/datepicker.android.js) and [`datepicker.ios.js`](https://github.com/gcanti/tcomb-form-native/blob/master/lib/templates/bootstrap/datepicker.ios.js)) use the native components provided by React Native: [DatePickerIOS](https://facebook.github.io/react-native/docs/datepickerios.html) and [DatePickerAndroid](https://facebook.github.io/react-native/docs/datepickerandroid.html).

`DatePickerIOS` is a wrapper around iOS [UIDatePicker](https://developer.apple.com/reference/uikit/uidatepicker). Despite that UIDatePicker has `locale` attribute, this is not exposed in the current implementation of RN's DatePickerIOS (`mode` is the only attribute exposed). Maybe you could open an issue into the RN repo.

`DatePickerAndroid` is a wrapper around [https://developer.android.com/reference/android/app/DatePickerDialog.html](https://developer.android.com/reference/android/app/DatePickerDialog.OnDateSetListener.html). There are some workarounds to change the language for the native component as explained [here](http://stackoverflow.com/questions/21257014/set-language-to-french-in-android-datepickerdialog) but they are not exposed by RN's component.

So in other words, with the current implementation of the datepickers provided by RN (that tcomb-form-native relies on) there is no way to change the language, you're stuck with the system language 😕

You can look for other datepicker components with localization support and create a tcomb-form-native factory to use them in your forms.

**Comment by javiercr** [comment]
I have opened a new feature request for React Native:
https://react-native.canny.io/feature-requests/p/localization-support-for-datepicker

**Comment by alterx** [comment]
@javiercr @michaelbenker , `DatePickerIOS` currently supports the `locale` prop as seen [here](https://facebook.github.io/react-native/docs/datepickerios#locale).

I've created a small PR to add support for it (#556)
